analysis_options.yaml

```diff
@@ -1,8 +1,8 @@
-include: package:pedantic/analysis_options.yaml
+include: package:flutter_lints/flutter.yaml
 
 analyzer:
   exclude:
-    - snowplow_dummy_server/*
+    - snowplow_dummy_server/**
```

Comment:
question: what is the diff between * and ** here?

Reply:
`**` is recursive, so it will match all files in `snowplow_dummy_server` AND in all child folders
cause with just `*`, it was still analyzing `snowplow_dummy_server/bin/main.dart`, but we want to exclude it